Dockerfile

```diff
@@ -0,0 +1,23 @@
+# syntax=docker/dockerfile:1.5
+FROM python:3.9-slim-bullseye
+ARG SDK_VERSION
+
+RUN <<EOF
+apt update --yes
+apt upgrade --yes
+apt install --yes --no-install-recommends git
+python -m pip install -U pip wheel
+python -m pip install orquestra-sdk[ray]==${SDK_VERSION}
+EOF
+
+RUN useradd -ms /bin/bash -d /home/orquestra orquestra --uid 1000 --gid 100
+USER 1000
+WORKDIR /home/orquestra
+
+ENV RAY_STORAGE=/tmp
```

Comment:
Are these workflow-driver specific env vars?

Reply:
AFAIK they're not. @SebastianMorawiec can confirm.

Reply:
ORQ_RAY_DOWNLOAD_GIT_IMPORTS is used in SDK overall, should be only set when running on remote.
ORQ_RAY_SET_TASK_RESOURCES was removed by me. Can be removed here as well I believe

Reply:
Sure, will remove.